int32 id
---
bool setsave_flash_accepted